config/jobs/cert-manager/release-previous/cert-manager-release-previous-presubmits.yaml

```diff
@@ -533,3 +533,60 @@ presubmits:
         options:
         - name: ndots
           value: "1"
+
+  - name: pull-cert-manager-e2e-v1-18
+    cluster: gke
+    context: pull-cert-manager-e2e-v1-18
+    always_run: true
```

Comment:
We don't want `always_run: true` along with `optional: true`

Reply:
Also: it looks like you didn't update this file by copying and pasting from `cert-manager-presubmits.yaml` in the parent directory as this same issue is still on the `v1-17` job? (I made the same mistake last time, which is why it's on that job and I think you've c+p'd that 😅)